.github/workflows/coverage.yml

```diff
@@ -15,9 +15,9 @@ jobs:
   coverageReport:
     strategy:
       matrix:
-        api-level: [21,30]
```

Comment:
hi  @gouri-panda , why we are removing api level 21 ?

Reply:
It's unnecessary now. Now we are testing this nightly, there no need to be double-checked.

Reply:
hi @kelson42 can you please confirm this ?

Reply:
i have no clue about the consequences and impact of such a decision. If you want me to take a decision, please let me know clearly what it implies.

Reply:
@kelson42 Most of the project happy with 1 API(we did that before with only API 21) . We can test all the different APIs in nightly.

Reply:
Api21 is the older API right? So this is corresponding to the oldes version of A ndroid we support i guess. How to you check that we don’t inadvertantly stop to support old versions of An droid devices if you remove CI with old API?

Reply:
@kelson42 We are checking with the nightly.